scripts/install_git_hooks.py

```diff
@@ -0,0 +1,76 @@
+# /// script
+# requires-python = ">=3.10"
+# dependencies = [
+#     "inquirer",
+#     "rich",
+# ]
+# ///
+# To run this script and install git hooks, run the following command:
+#
+#   uv run scripts/install_git_hooks.py
+#
+from pathlib import Path
+
+from inquirer.shortcuts import list_input
+from rich import print as pprint
+
+HOOK_BODY = """
+#!/usr/bin/env bash
+
+echo "🧹 Running formatting...\n"
+uv run ruff format --check
+
+if [ $? -ne 0 ]
+then
+    echo "⚠ Formatting failed. Running autofix & aborting..."
+    uv run ruff format
+    exit 1
```

Comment:
IMO, it would be better to exit at the end, after all of the checks have been run

Reply:
Initially I had it like this, but mypy feels really slow - so I decided that it's better experience to fail fast in case of format / linting issues

Reply:
ok, makes sense